src/components/Tile.js

```diff
@@ -0,0 +1,42 @@
+import { useGameContext } from '../context/GameContext.js';
+import './Tile.css';
+// import horse from '../../public/horse.png';
+// import snake from '../../public/snake.png';
+export default function Tile({ tileContent, index }) {
+  const { player, setPlayer, board, active, setBoard, setMessage } = useGameContext();
+
+  function handleClick() {
+    if (tileContent === 'Snake') {
+      return;
+    }
+    if (tileContent === 'Horse') {
+      return;
+    }
+    if (!active) {
```

Comment:
this could be a one-liner: ```if (!active) return``` — you only need the ```{}``` if you need to have multiple lines of code that execute in the if block